scaaml/capture/ascad/ascad_capture.py

```diff
@@ -17,10 +17,12 @@
 from scaaml.capture.aes.aes_capture_contexts import capture_aes_dataset
 from scaaml.capture.scope import PicoScope
 
+from typing import Any, Dict, Optional
+
 
 def capture_ascad_default_parameters(
         firmware_sha256: str,
-        crypto_implementation=AESSBOX,
+        crypto_implementation: Any = AESSBOX,
```

Comment:
NIT below we have the following (again with lower case `type`):
```suggestion
        crypto_implementation: type[AESSBOX] = AESSBOX,
```

Reply:
Done